engines/sci/detection.cpp

```diff
@@ -784,6 +785,8 @@ SaveStateList SciMetaEngine::listSaves(const char *target) const {
 	return saveList;
 }
 
+bool SciMetaEngine::simpleSaveNames() const { return true; }
```

Comment:
To elaborate, the SCI engine could give a list of all savegames (and other saved files) for the current game, but it won't be limited to the usual `target.###`.